src/templates/static/_layout/footer.jsx

```diff
@@ -199,66 +199,70 @@ const FooterNormal = () => (
                 </div>
             </div>
         </div>
-        <div id='footer-regulatory' className='primary-bg-color-dark gr-padding-10 gr-child'>
+        <div id='footer-regulatory' className='primary-bg-color-dark gr-padding-10'>
             <div className='container'>
                 <div className='gr-row'>
-                    <p id='regulatory-text' className='gr-9 gr-12-m gr-7-p gr-padding-10 gr-parent content-inverse-color no-margin'>
-                        {it.L(
-                            'In the EU, financial products are offered by Binary Investments (Europe) Ltd., Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta, licensed and regulated as a Category 3 Investment Services provider by the Malta Financial Services Authority (license no. IS/70156). In the Isle of Man and the UK, Volatility Indices are offered by Binary (IOM) Ltd., First Floor, Millennium House, Victoria Road, Douglas, IM2 4RW, Isle of Man, British Isles; licensed and regulated respectively by (1) the Gambling Supervision Commission in the Isle of Man (current licence issued on 31 August 2017) and by (2) the Gambling Commission in the UK ([_1]). In the rest of the EU, Volatility Indices are offered by Binary (Europe) Ltd., Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta; licensed and regulated by (1) the Malta Gaming Authority in Malta (licence no. [_2] issued on 01 August 2018), for UK clients by (2) the UK Gambling Commission ([_3]), and for Irish clients by (3) the Revenue Commissioners in Ireland (Remote Bookmaker\'s Licence no. 1010285 issued on 1 July 2017). View complete [_4]. Some products are not available in all countries. This website\'s services are not made available in certain countries such as the USA, Canada, Costa Rica, Hong Kong, or to persons under age 18.',
-                            `<a href="https://secure.gamblingcommission.gov.uk/PublicRegister/Search/Detail/39172" target="_blank">${it.L('license reference no: 39172')}</a>`,
-                            'MGA/B2C/102/2000',
-                            `<a href="https://secure.gamblingcommission.gov.uk/PublicRegister/Search/Detail/39495" target="_blank">${it.L('license reference no: 39495')}</a>`,
-                            `<a href=${it.url_for('regulation')}>${it.L('Regulatory Information')}</a>`,
-                        )}
-                    </p>
-                    <div className='gr-3 gr-12-m gr-5-p center-text' id='social-networks'>
-                        <div className='gr-row gr-row-align-center-m gr-padding-10 gr-parent id-hide'>
-                            <div className='gr-5 gr-6-m'>
-                                <a href='https://www.gov.im/gambling/' target='_blank' rel='noopener noreferrer'>
-                                    <img id='iom_icon_footer' className='responsive' src={it.url_for('images/pages/footer/isle-of-man.png')} />
+                    <div className='gr-12'>
```

Comment:
i think iom & gamstop logo needs to be hidden for ID.. we have `id-hide` class to hide the contents